store/json.go

```diff
@@ -195,7 +197,7 @@ func (kvs *jsonFileStore) Lock(timeout time.Duration) error {
 		return errors.Wrap(err, "processLock acquire error")
 	}
 
-	log.Printf("Acquired process lock")
+	log.Printf("Acquired process lock with timeout value of %v ", timeout)
```

Comment:
can you add unit for timeout in log

Reply:
I have tested the log it will print as 1m if it was less than a minute it will show 30s.